File: GobsProgram.lua
local line = ""
for i = 1, 5 do
  line = line .. "Penus  "
end
line = line .. "Penus"
local s = nil
while s ~= "n" and s~= "no" do
  print("Gob's Program:  Y/N?")
  io.write("? ")
  s = string.lower(io.read("*l"))
  if s == "y" or s == "yes" then
    while true do
      print(line)
    end
  end
end

File: gobsprogram.rs
use std::io;
use std::io::Write;
use std::process;

fn main() {

    let mut ans = String::new();
    let mut i: u32 = 0;

    loop{
        print!("Gob`s Program:  Y/N?\n?  ");
        let _ = io::stdout().flush();
        io::stdin().read_line(&mut ans).expect("");

        match ans.trim() {
            "Y" | "Yes" => loop{
                i = i + 1;
                print!("Penus");
                if i % 7 != 0 { print!("  ") } else { print!("\n") }
            },
            "N" | "No" => process::exit(0),
            _  => continue
        }
    }
}
